frontend_tests/node_tests/message_flags.js

```diff
@@ -69,6 +69,35 @@ run_test("starred", (override) => {
         starred: false,
     });
 });
+
+run_test("starring local echo", () => {
+    let ui_updated = false;
+    ui.update_starred_view = () => {
+        ui_updated = true;
+    };
```

Comment:
Same question about whether we should be using `override` or other newer mocking technology here.

Reply:
We should definitely use `override()` here.